### DISCUSSION TITLE
template: initrd.img:195: bad character U+001D

### DISCUSSION THREAD

**Discussion by Yonyc** [photoprism/photoprism#3042] [Q&A] [answered]
Hello everyone,

I have a problem with my photoprism installation and I don't know how to fix it.
When I start the service, it shows me no errors. But, when I try to connect, I have a connection refused.
So I've checked the logs and the only error I got is `level=error msg="template: initrd.img:195: bad character U+001D"`

Then I tried to launch the server in debug mod and I got this.
`time="2022-12-30T16:45:48+01:00" level=debug msg="server: template: initrd.img:195: bad character U+001D (GET /library/login HTTP/1.1\r\nHost: 192.168.0.9:2342\r\nAccept: text/html,application/xhtml+xml,application/xml;q=0.9,image/avif,image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3;q=0.9\r\nAccept-Encoding: gzip, deflate\r\nAccept-Language: en-US,en;q=0.9,fr;q=0.8,la;q=0.7\r\nConnection: keep-alive\r\nUpgrade-Insecure-Requests: 1\r\nUser-Agent: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/108.0.0.0 Safari/537.36\r\n\r\n)\n/usr/local/go/src/html/template/template.go:368 (0xa71bdb)\n\tMust: panic(err)\n/root/go/pkg/mod/github.com/gin-gonic/gin@v1.8.1/render/html.go:71 (0xa719f9)\n\tHTMLDebug.loadTemplate: return template.Must(template.New(\"\").Delims(r.Delims.Left, r.Delims.Right).Funcs(r.FuncMap).ParseFiles(r.Files...))\n/root/go/pkg/mod/github.com/gin-gonic/gin@v1.8.1/render/html.go:61 (0xa74517)\n\tHTMLDebug.Instance: Template: r.loadTemplate(),\n/root/go/pkg/mod/github.com/gin-gonic/gin@v1.8.1/context.go:919 (0xab2417)\n\t(*Context).HTML: instance := c.engine.HTMLRender.Instance(name, obj)\n/tmp/photoprism/internal/server/routes_static.go:43 (0xe088b8)\n\tregisterStaticRoutes.func3: c.HTML(http.StatusOK, conf.TemplateName(), values)\n/root/go/pkg/mod/github.com/gin-gonic/gin@v1.8.1/context.go:173 (0xe067b9)\n\t(*Context).Next: c.handlers[c.index](c)\n/tmp/photoprism/internal/server/logger.go:20 (0xe0679c)\n\tLogger.func1: c.Next()\n/root/go/pkg/mod/github.com/gin-gonic/gin@v1.8.1/context.go:173 (0xe06aca)\n\t(*Context).Next: c.handlers[c.index](c)\n/tmp/photoprism/internal/server/recovery.go:32 (0xe06ab4)\n\tRecovery.func1: c.Next()\n/root/go/pkg/mod/github.com/gin-gonic/gin@v1.8.1/context.go:173 (0xab9a70)\n\t(*Context).Next: c.handlers[c.index](c)\n/root/go/pkg/mod/github.com/gin-gonic/gin@v1.8.1/gin.go:616 (0xab96d8)\n\t(*Engine).handleHTTPRequest: c.Next()\n/root/go/pkg/mod/github.com/gin-gonic/gin@v1.8.1/gin.go:572 (0xab921c)\n\t(*Engine).ServeHTTP: engine.handleHTTPRequest(c)\n/usr/local/go/src/net/http/server.go:2947 (0x7c638b)\n\tserverHandler.ServeHTTP: handler.ServeHTTP(rw, req)\n/usr/local/go/src/net/http/server.go:1991 (0x7c15a6)\n\t(*conn).serve: serverHandler{c.server}.ServeHTTP(w, w.req)\n/usr/local/go/src/runtime/asm_amd64.s:1594 (0x473e80)\n\tgoexit: BYTE\t$0x90\t// NOP\n"`

Does anyone have any idea of what this error means ?
Thank you

**Comment by lastzero** [comment, marked as answer]
This should be fixed in our developer preview build:
- #2946

**Reply by Yonyc** [reply]
Thank you a lot man.
Have a nice weekend
